canvas_sdk/v1/data/task.py

```diff
@@ -99,10 +98,45 @@ class Meta:
     task_label = models.ForeignKey(TaskLabel, on_delete=models.DO_NOTHING, null=True)
     task = models.ForeignKey(Task, on_delete=models.DO_NOTHING, null=True)
 
+class NoteTask(Model):
+    """Note Task."""
+
+    class Meta:
+        db_table = "canvas_sdk_data_api_notetask_001"
+
```

Comment:
you'll need to add the following missing fields (which all commands have, and you included in your database view query!):
```
created = models.DateTimeField(auto_now_add=True)
modified = models.DateTimeField(auto_now=True)
originator = models.ForeignKey("v1.CanvasUser", on_delete=models.DO_NOTHING, related_name="+")
committer = models.ForeignKey("v1.CanvasUser", on_delete=models.DO_NOTHING, null=True, related_name="+")
deleted = models.BooleanField(default=False)
entered_in_error = models.ForeignKey("v1.CanvasUser", on_delete=models.DO_NOTHING, null=True, related_name="+")
```